docs/conf.py

```diff
@@ -71,6 +71,52 @@
 #
 # source_encoding = 'utf-8-sig'
 
+# Add link to Binder in Prolog (WRITE MORE DETAILS ONCE FIXED)
+# -- Get version information  ----------------------------
+try:
+    from subprocess import check_output
+    release = qcodes.__version__
+
+    if 'rc' in release:
+        release = release.split('+')[0]
```

Comment:
This seems very specific to rcs. What about alphas or betas? I think it is better to use something like packageing https://packaging.pypa.io/en/latest/version/ to extract the part of the version that we want